roles/partition-disk/meta/main.yml

```diff
@@ -0,0 +1,50 @@
+galaxy_info:
+  author: Satellite QE Team
+  description: Satellite QE Team
+  company: RedHat
```

Comment:
Red Hat

Reply:
Argh! I knew about this, my bad will fix it.